SPECS/ocaml-cinaps/ocaml-cinaps.spec

```diff
@@ -58,7 +54,7 @@ mkdir -p %{buildroot}%{_mandir}/man1
 help2man -N --version-string=%{version} %{buildroot}%{_bindir}/cinaps > \
   %{buildroot}%{_mandir}/man1/cinaps.1
 
-%if %{with test}
+%if %{with_check}
```

Comment:
You can drop `%if %{with_check}` and `%endif` completely. The check is only ran during tests builds anyway.

Reply:
Removed